.gitignore

```diff
@@ -1,3 +1,4 @@
 launch.json
 .vscode
 compile
+builddir
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

:)